cloud/cluster/fetch_cron_test.go

```diff
@@ -0,0 +1,81 @@
+package cluster_test
+
+import (
+	"testing"
+	"github.com/scootdev/scoot/cloud/cluster"
+	"time"
+	"reflect"
+	"sync"
+	"sort"
+	// "fmt"
+)
+
+func TestFetchCron(t *testing.T) {
+	h := makeCronHelper(t)
+	h.assertFetch(t, h.setupTest("host1:1234"))
```

Comment:
I think this test might feel easier if you just wait for the expected states on the channel, not try to wait for it to wind its way through cluster, e.g.